TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/Claw.java

```diff
@@ -29,7 +29,7 @@ public boolean run(@NonNull TelemetryPacket packet) {
         }
     }
 
-    public Action openAction() {
+    public OpenClaw openAction() {

```

Comment:
any reason for not keeping to the generic Action return type? I feel like that would be preferable for consistency purposes